src/menu.c

```diff
@@ -102,6 +102,11 @@ char *extract_urls(const char *to_match)
  */
 void open_browser(const char *in)
 {
+        if (!settings.browser_cmd) {
+                LOG_C("Unable to open browser: No browser command set.");
```

Comment:
I wouldn't conclude by "The program cannot continue to work." that `dunst` has to bail out. At this point, we cannot continue to work, because we can't start a fallback browser. In my conclusion this would justify a critical.